pkg/products/grafana/reconciler.go

```diff
@@ -542,3 +555,53 @@ func buildGrafanaDashboardStrings(softLimits []uint32) (string, string) {
 
 	return graphQueries, dashboardVariables
 }
+
+func (r *Reconciler) reconcileConsoleLink(ctx context.Context, serverClient k8sclient.Client, installation *integreatlyv1alpha1.RHMI) error {
```

Comment:
It's being done in multiple locations in this file so feel free to ignore but the installation object is available in the the Reconciler so could be a bit neater to use it rather than passing. Feel free to ignore tho as it's done elsewhere.

Reply:
removed installation param.